src/components/controls/Dropdown.tsx

```diff
@@ -3,50 +3,24 @@ import { observer } from 'mobx-react';
 import * as s from './dropdown.scss';
 
 export interface IDropdownItem {
-    value: string|number;
+    value: string;
     label: string;
 }
 
-interface IDropdownBaseProps {
+interface IDropdownProps {
     label?: string;
     selected?: string;
     disabled?: boolean;
+    items: IDropdownItem[];
```

Comment:
items: ICodeListItem[]

(in the future, if we need custom codeLists that can't be queried from backend, we can add them to codeListStore)

Reply:
(also remove mapping from ICodeListItem[] to IDropDownItem[] at codeListStore)